frontend/src/app/shared/components/modals/save-modal/save-query.modal.ts

```diff
@@ -92,6 +92,8 @@ export class SaveQueryModalComponent extends OpModalComponent {
   }
 
   public saveQueryAs($event:JQuery.TriggeredEvent) {
+    $event.preventDefault();
```

Comment:
I stumbled upon this by testing. The regular form is still submitted and causes a navigation change, but that is usually cancelled by ui-router.